src/components/dialogs/network-modifications/hvdc-line/lcc/creation/filters-mcs-table.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { Box, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@mui/material';
+import Tooltip from '@mui/material/Tooltip';
+import IconButton from '@mui/material/IconButton';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import { useIntl } from 'react-intl';
+import { useCallback, useMemo, useState } from 'react';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useFieldArray } from 'react-hook-form';
+import {
+    COUNT,
+    FILTERS_MCS_TABLE,
+    MAX_Q_AT_NOMINAL_V,
+    MCS_SELECTED,
+    SHUNT_COMPENSATOR_ID,
+    SHUNT_COMPENSATOR_NAME,
+} from '../../../../../utils/field-constants';
+import { FloatInput, SwitchInput, TextInput } from '@gridsuite/commons-ui';
+import { ReactivePowerAdornment } from '../../../../dialog-utils';
+
+interface FiltersMcsTableProps {
```

Comment:
```suggestion
interface FiltersShuntCompensatorTableProps {
```
To change all Msc to ShuntCompensator in other files